gmail/src/services/log_email.ts

```diff
@@ -0,0 +1,18 @@
+import { postJsonRpc } from "../utils/http";
+import { URLS } from "../const";
+import { State } from "../models/state";
+
+/**
+ * Log the given email body in the chatter of the given record.
+ */
+export function logEmail(recordId: number, recordModel: string, emailBody: string): boolean {
```

Comment:
And that could be a mixin for the "loggable models" that adds a "logEmail" method on each of them.
Wouldn't that be nice? :smile: But I'm thinking too much "Odoo" probably.

Reply:
It will add to much complexity I think... (I will need to add an attribute on the class for the model name, etc)